### ISSUE TITLE
pip install peewee failed

### ISSUE THREAD

**Issue by openmartin** [coleifer/peewee#761]
(1.0.0)ubuntu@localhost:/home/ubuntu/v_env$ pip install peewee
Collecting peewee
  Using cached peewee-2.7.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-M40xkD/peewee/setup.py", line 42, in <module>
        scripts = ['pwiz.py'],
      File "/usr/lib/python2.7/distutils/core.py", line 137, in setup
        ok = dist.parse_command_line()
      File "/opt/test/crazzkeyboard/1.0.0/local/lib/python2.7/site-packages/setuptools/dist.py", line 299, in parse_command_line
        result = _Distribution.parse_command_line(self)
      File "/usr/lib/python2.7/distutils/dist.py", line 467, in parse_command_line
        args = self._parse_command_opts(parser, args)
      File "/opt/test/crazzkeyboard/1.0.0/local/lib/python2.7/site-packages/setuptools/dist.py", line 601, in _parse_command_opts
        nargs = _Distribution._parse_command_opts(self, parser, args)
      File "/usr/lib/python2.7/distutils/dist.py", line 523, in _parse_command_opts
        cmd_class = self.get_command_class(command)
      File "/opt/test/crazzkeyboard/1.0.0/local/lib/python2.7/site-packages/setuptools/dist.py", line 425, in get_command_class
        if command in self.cmdclass:
    TypeError: argument of type 'NoneType' is not iterable

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-M40xkD/peewee

**Comment by openmartin** [comment]
on Ubuntu 14.04.2 LTS
python 2.7.6

**Comment by alexanderad** [comment]
I confirm we got this issue this morning on our CI.
Out trace is:
```python
Downloading/unpacking peewee>=2.6.4 (from imedicare.shared==0.3.68->-r /home/u877/src/repo/app-api/requirements.txt (line 7))
  Running setup.py (path:/home/u877/python-env/build/peewee/setup.py) egg_info for package peewee
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/home/u877/python-env/build/peewee/setup.py", line 42, in <module>
        scripts = ['pwiz.py'],
      File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/core.py", line 138, in setup
        ok = dist.parse_command_line()
      File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 250, in parse_command_line
        result = _Distribution.parse_command_line(self)
      File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/dist.py", line 467, in parse_command_line
        args = self._parse_command_opts(parser, args)
      File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 534, in _parse_command_opts
        nargs = _Distribution._parse_command_opts(self, parser, args)
      File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/dist.py", line 523, in _parse_command_opts
        cmd_class = self.get_command_class(command)
      File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 359, in get_command_class
        if command in self.cmdclass:
    TypeError: argument of type 'NoneType' is not iterable
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/home/u877/python-env/build/peewee/setup.py", line 42, in <module>

    scripts = ['pwiz.py'],

  File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/core.py", line 138, in setup

    ok = dist.parse_command_line()

  File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 250, in parse_command_line

    result = _Distribution.parse_command_line(self)

  File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/dist.py", line 467, in parse_command_line

    args = self._parse_command_opts(parser, args)

  File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 534, in _parse_command_opts

    nargs = _Distribution._parse_command_opts(self, parser, args)

  File "/usr/local/python/cpython-2.7/lib/python2.7/distutils/dist.py", line 523, in _parse_command_opts

    cmd_class = self.get_command_class(command)

  File "/home/u877/python-env/lib/python2.7/site-packages/setuptools/dist.py", line 359, in get_command_class

    if command in self.cmdclass:

TypeError: argument of type 'NoneType' is not iterable

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/u877/python-env/build/peewee
Storing debug log for failure in /home/u877/.pip/pip.log
[exit status: 1]```

**Comment by klen** [comment]
:+1: Python 3.4.2 https://travis-ci.org/klen/muffin-admin/builds/92235523

Looks like setup with peewee 2.7.0 is broken.

**Comment by tfeldmann** [comment]
Same error occurs on Mac OS X 10.11, Python 3.5

**Comment by coleifer** [comment]
This should be fixed with the 2.7.1 release. Tested with and without cython on py 2.7.9.
